recipes-robot/fs-overlay/ot3-overlay-mount-points_1.0.bb

```diff
@@ -0,0 +1,18 @@
+SUMMARY = "ot3 overlay mount point directory structure"
+LICENSE = "CLOSED"
+BB_STRICT_CHECKSUM = "0"
+PV = "1.0"
+
+S = "${WORKDIR}"
+
+inherit allarch
+
+do_install () {
+        install -d ${D}/data/upper/
```

Comment:
Does this snippet mean that the _entirety_ of `/etc` (and `/var`, but that doesn't matter as much) will be overlayed with a writable layer? Because we definitely do not want that - then we wouldn't be able to update the contents of `/etc`, because it might be overlayed.